Item/Item.module.less

```diff
@@ -47,11 +47,13 @@
 		.agate-item-sizing();
 
 		line-height: @agate-item-line-height;
+		max-width: @agate-item-max-width;
```

Comment:
Could we remove this property? I think it's too restrict even it's on the guide.